core/src/main/resources/filodb-defaults.conf

```diff
@@ -211,6 +211,9 @@ filodb {
   shard-manager {
     # Minimum time required between successive automatic shard reassignments done by ShardManager
     reassignment-min-interval = 2 hours
+
+    # Time between each TenantIngestionMetering query/publish job
+    metering-query-interval = 15 minutes
```

Comment:
Move this config as a sibling of `shard-key-level-ingestion-metrics-enabled` config

Reply:
Done!